tests/Set/Symfony43/Fixture/event_fixture.php.inc

```diff
@@ -17,8 +15,6 @@ class SomeEventListener
 {
     public function someMethod(\Symfony\Component\HttpKernel\Event\ExceptionEvent $exceptionEvent)
     {
-        $exception = $exceptionEvent->getThrowable();
-        $exceptionEvent->setThrowable($exception);
     }
 }
 
```

Comment:
This file can be removed completelly, as it does not show any change now.

Reply:
No, the class is changed.

Reply:
Ah, I see :+1: